go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/go-asn1-ber/asn1-ber
+module gopkg.in/asn1-ber.v1
 
-go 1.13
+go 1.21
```

Comment:
If there's a way to not have to bump this I'd prefer that.

Reply:
sure, the main reason I did this was to be able to use this in a go.work file